src/main/webapp/js/createGame.js

```diff
@@ -106,5 +169,66 @@ function getActiveStageElement() {
 * @return {string} the first className of the element which should be in the form of "stage" + [the stage number]
 */
 function getStageNumber(stageElement) {
-    return stageElement.classList[0];
+  return stageElement.classList[0];
+}
+
+/**
+* Gets the data from the form on the createGame.html page and sends it to the server
+*/
+function getDataFromForm() {
+  const numStages = document.getElementById('stages').getElementsByTagName('input').length;
+  var stageKeys = [];
+  var stageSpawnLocations = []; // ex: [{'latitude': 1, 'longitude':2}, {'latitude': 3, 'longitude': 4}]
+  var stageStarterHints = [];
+  var hintLocations = []; // 2d array [[{'latitude': 1, 'longitude':2},{'latitude': 1, 'longitude':2}], [{'latitude': 1, 'longitude':2}]]
+  var hintTexts = [];
+
+  // Stages and hints are 1 indexed
+  for (var count = 1; count <= numStages; count++) {
+    var stage = document.getElementById('stage' + count + 'Hints');
+    var starterPos = stage.querySelector('#starter-position').value;
+    starterPos = starterPos.replace(")", "").replace("(", "").replace(" ", "").split(",");
+
+    var dict = {};
+    dict['latitude'] = starterPos[0];
+    dict['longitude'] = starterPos[1];
+    if (isNaN(starterPos[0]) || isNaN(starterPos[1])) {
+      window.alert("Input for latitude and longistude must be numbers! In format (123, 456) or 123, 456");
```

Comment:
longitude*

Reply:
I can't spell still lol. I will change this!